Documentation/config/core.txt

```diff
@@ -79,6 +83,13 @@ core.fsmonitorHookVersion::
 	something that can be used to determine what files have changed
 	without race conditions.
 
+core.useBuiltinFSMonitor::
+	Enable the built-in filesystem event watcher. This requires a
```

Comment:
Maybe embellish this a bit ??

```
If set to true, ignore the `core.fsmonitor` hook and enable the built-in filesystem
event watcher linkgit:git-fsmonitor--daemon[1].
+
Like external (hook-based) FSMonitors, the built-in FSMonitor can speed up Git
commands, such `git status`, that need to refresh the Git index in a worktree 
with many files.  The built-in FSMonitor facility eliminates the need to install and
maintain external third-party monitoring tool.
+
The built-in FSMonitor is currently available only on a limited set of supported
platforms.
```

Reply:
I reworded this only slightly, and replaced the previous version.